lib/chrome.js

```diff
@@ -58,6 +59,16 @@ class Events {
 
 const MIN_INTERVAL = 1000;
 
+function keyToId(key) {
+  let md5Str = md5(key);
+  const parts = [];
+  [8,4,4,4,12].map(numChars => {
```

Comment:
I'll leave it with `[8,4,4,4,12].map` unless we find a nodejs module that does exactly that.